### DISCUSSION TITLE
lobe-chat是否能变成网站客服AI?

### DISCUSSION THREAD

**Discussion by wzcheng99** [lobehub/lobehub#1550] [General]
我需要构建一个客服AI，lobe-chat是否能支持这样的UI么，另外是否能支持RAG等等

**Comment by arvinxx** [comment]
暂不考虑

**Comment by cy948** [comment]
这样的需求可以去试试 [fastgpt](https://github.com/labring/FastGPT)

**Comment by cy948** [comment]
&nbsp;fastgpt 会更合适。
